frontend/Components/ListingCard/ListingCard.module.css

```
@@ -0,0 +1,41 @@
+.cardContainer {
+  width: 350px;
+}
+
+.cardContent {
+  display: flex;
+  flex-direction: column;
+  gap: 1.3em;
+}
+
+.cardHeader {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+  white-space: nowrap;
+}
+
+.headerSubtitle {
+  font-size: 16px;
+  font-style: italic;
+  margin-bottom: 0;
+}
+
+.headerTitle {
+  font-weight: 700;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  width: 200px;
+}
+
+.cardPrice {
```

Ya que usamos CSS modules, esto se puede llamar:

```suggestion
.price {
```